tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Validators/ApplicationUserValidatorTests.cs

```diff
@@ -0,0 +1,11 @@
+﻿using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.UnitTests.Validators
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.All)]
+    public sealed class ApplicationUserValidatorTests
```

Comment:
fantastic tests!

Reply:
Wow they are the best kind. Updated to include some actual tests.